node/src/main/scala/com/wavesplatform/database/Caches.scala

```diff
@@ -122,18 +122,23 @@ abstract class Caches(spendableBalanceChanged: Observer[(Address, Asset)]) exten
   override def assetScript(asset: IssuedAsset): Option[(Script, Long)] = assetScriptCache.get(asset)
 
   private var lastAddressId = loadMaxAddressId()
-  protected def loadMaxAddressId(): BigInt
+  protected def loadMaxAddressId(): Long
```

Comment:
```suggestion
  protected def loadMaxAddressId(): AddressId
```

Reply:
This is actually a number, not a real address ID. It's only used in arithmetic operations when new address IDs are calculated, so I thought I'd leave it as is.